MatrixMultiplication/MatrixMultiplication/MatrixBenchmark.cs

```diff
@@ -0,0 +1,175 @@
+            var sequentialTimeArray = new double[numExperiment];
+
+            for (int j = 0; j < numExperiment; j++)
+            {
+                var matrix1 = CreateMatrix(benchmarkingSize[i].Item1, benchmarkingSize[i].Item2, -100, 100);
+                var matrix2 = CreateMatrix(benchmarkingSize[i].Item2, benchmarkingSize[i].Item1, -100, 100);
+
+                var sw = Stopwatch.StartNew();
+                parallelMultiplier.Multiply(matrix1, matrix2);
+                sw.Stop();
+                parallelTimeArray[j] = sw.ElapsedMilliseconds;
+
+                sw.Restart();
+                sequentialMultiplier.Multiply(matrix1, matrix2);
+                sw.Stop();
+                sequentialTimeArray[j] = sw.ElapsedMilliseconds;
+            }
+
+            var parallelMean = CalculateMean(parallelTimeArray.Sum(), numExperiment);
+            var sequentialMean = CalculateMean(sequentialTimeArray.Sum(), numExperiment);
+
+            var parallelStandardDeviation = CalculateStandardDeviation(parallelTimeArray, parallelMean);
+            var sequentialStandardDeviation = CalculateStandardDeviation(sequentialTimeArray, sequentialMean);
+
+            Console.WriteLine(
+            $"{"Parallel",-10} " +
+            $"{$"{benchmarkingSize[i].Item1}x{benchmarkingSize[i].Item2}",-10} " +
+            $"{parallelMean,-15:F2} " +
+            $"{numExperiment,-15} " +
+            $"{parallelStandardDeviation,-20:F4}");
+
+            Console.WriteLine(
+            $"{"Sequential",-10} " +
+            $"{$"{benchmarkingSize[i].Item1}x{benchmarkingSize[i].Item2}",-10} " +
+            $"{sequentialMean,-15:F2} " +
+            $"{numExperiment,-15} " +
+            $"{sequentialStandardDeviation,-20:F4}");
+        }
+    }
+
+    /// <summary>
+    /// Create matrix by parameters.
+    /// </summary>
+    /// <param name="rowsNum">Number of rows.</param>
+    /// <param name="columnsNum">Number of columns.</param>
+    /// <param name="minNum">Min border of value.</param>
+    /// <param name="maxNum">Max border of value.</param>
+    /// <exception cref="ArgumentException">Argument exception.</exception>
+    /// <returns>Matrix.</returns>
+    public static int[,] CreateMatrix(int rowsNum, int columnsNum, int minNum, int maxNum)
+    {
+        if (rowsNum <= 0 || columnsNum <= 0 || minNum > maxNum)
+        {
+            throw new ArgumentException();
+        }
+
+        var matrix = new int[rowsNum, columnsNum];
+
+        for (int i = 0; i < rowsNum; i++)
+        {
+            for (int j = 0; j < columnsNum; j++)
+            {
+                matrix[i, j] = Random.Shared.Next(minNum, maxNum + 1);
+            }
+        }
+
+        return matrix;
+    }
+
+    /// <summary>
+    /// Write resultMatrix to file.
+    /// </summary>
+    /// <param name="matrix">ResultMatrix.</param>
+    public static void WriteFile(int[,] matrix)
+    {
+        using (StreamWriter writer = new StreamWriter("resultMatrix.txt"))
```

Comment:
Для таких случае используйте using declaration
```suggestion
        using StreamWriter writer = new StreamWriter("resultMatrix.txt");
```